LanguageFeatures/Control-flow-collections/scoping_A01_t01.dart

```diff
@@ -36,4 +36,11 @@ main() {
       for (var j = i; j < 4; j++) i + j,
   ];
   Expect.listEquals([2, 3, 4, 4, 5], list3);
+  
+  var x = "a";
+  List<String> list4 = [
+    for (var x in ["b", "c"]) x,
+    x
+  ];
+  Expect.listEquals(["b", "c", "a"]);  
```

Comment:
Should be 
Expect.listEquals(["b", "c", "a"], list4);

Please fix at your earliest convinience, maybe added to another PR. Thanks.